docs/SUMMARY.md

```diff
@@ -13,14 +13,9 @@
 ## Core features
 
 * [Overview](core-features/fhe_basics.md)
-* [Table lookups](core-features/table_lookups.md)
+* [Table lookups (basics)](core-features/table_lookups_basics.md)
```

Comment:
Why `basics? We can have two `Table lookups` in two different sections IMO. I don't think `basics` is necessary.

Reply:
I am afraid that, we may end up 
- renaming the .md (because Yuxi likes the md names to be the same as entries)
- and then, overwrite the other table_lookups.md

Reply:
@umut-sahin , can we merge?

Reply:
So @umut-sahin : 
- I renamed the "basic" one into table_lookups.md
- and the other one to table_lookups_advanced.md

Reply:
Good now @umut-sahin ?